Sources/Rownd/Views/BottomSheetController.swift

```diff
@@ -66,6 +66,25 @@ class BottomSheetController: UIViewController {
         super.viewDidDisappear(animated)
         self.controller = nil
     }
+    
+    func updateBottomSheetHeight(_ number: CGFloat) {
+        if let sheetController = sheetController {
+            guard let controller = self.controller else {
+                return
+            }
+            DispatchQueue.main.async {
+                let hubViewController = controller as! HubViewController
```

Comment:
This should probably use conditional downcasting `as?` and check for `nil` just in case.

Reply:
Checks for nil have been added